### PR TITLE
[ONNX] Export clamp for opset 11

### PR DESCRIPTION
**Pull request by BowenBao**
- Export clamp for opset 11, which enables dynamic min/max inputs.
- Bump ONNX Runtime version in CI to enable opset 11 onnx::clip tests.
~~- Re-enable some disabled tests, now that backend impl & fixes are in.~~

### CODE REVIEW

File: test/onnx/test_pytorch_onnx_onnxruntime.py [hunk 1]
```diff
@@ -979,10 +1028,6 @@ def setup_rnn_tests():
                 ('lstm', 'lstm', {}),
                 ('gru', 'gru', {})
         ):
-            # This is a hack to skip elman_rnn bidirectional tests for now
-            # TODO: Revert this once elman_rnn bidirectional issue is fixed
-            if base == 'elman' and bidirectional[1] == 'bidirectional':
```
Comment (houseroad) [inline test/onnx/test_pytorch_onnx_onnxruntime.py]: Why do we combine two changes together?

Comment (BowenBao) [reply test/onnx/test_pytorch_onnx_onnxruntime.py]: 1. for enabling disabled tests, it is because clamp tests needs ort version bump, and with new versions some disabled tests can be enabled now.
2. for enabling flaky lstm/gru/rnn tests, it's because the fix for relaxing tolerance is merged.

Both changes are small so I included them here, I'm ok with removing them and submit a separate PR.

Comment (houseroad) [reply test/onnx/test_pytorch_onnx_onnxruntime.py]: Yes, please submit a separate PR for enabling tests, thanks.

Comment (BowenBao) [reply test/onnx/test_pytorch_onnx_onnxruntime.py]: removed test changes from this one.

File: torch/onnx/symbolic_opset11.py [hunk 2]
```diff
@@ -11,13 +11,28 @@
 # This file exports ONNX ops for opset 11
 
 black_listed_operators = [
-    "eq", "ne", "scatter", "clamp", "clamp_min", "clamp_max", "sort", "topk", "hardtanh"
```
Comment (houseroad) [inline torch/onnx/symbolic_opset11.py]: Do we need to add clamp_min/max?

If not, did we accidentally put them in the black list?

**Review by houseroad — APPROVED** [review]
Looks good.

File: torch/onnx/symbolic_opset11.py (hunk 2 above)
Comment (BowenBao) [reply torch/onnx/symbolic_opset11.py]: We don't need to, they are helper functions called only by 'clamp' itself.

**Comment by houseroad** [comment]
Sep 16 18:56:52 self = <onnxruntime.capi.session.InferenceSession object at 0x7ff62bf677b8>
Sep 16 18:56:52 output_names = ['9', '10', '31']
Sep 16 18:56:52 input_feed = {'0': array([[[ 9.3826073e-01,  4.8886982e-01],
Sep 16 18:56:52         [-6.7308992e-01,  8.7283123e-01],
Sep 16 18:56:52         [ 1.0553575e+00,  1....  2.3025165e+00],
Sep 16 18:56:52         [-1.8816892e+00, -4.9727023e-02],
Sep 16 18:56:52         [-1.0449786e+00, -9.5650083e-01]]], dtype=float32)}
Sep 16 18:56:52 run_options = None
Sep 16 18:56:52 
Sep 16 18:56:52     def run(self, output_names, input_feed, run_options=None):
Sep 16 18:56:52         """
Sep 16 18:56:52         Compute the predictions.
Sep 16 18:56:52     
Sep 16 18:56:52         :param output_names: name of the outputs
Sep 16 18:56:52         :param input_feed: dictionary ``{ input_name: input_value }``
Sep 16 18:56:52         :param run_options: See :class:`onnxruntime.RunOptions`.
Sep 16 18:56:52     
Sep 16 18:56:52         ::
Sep 16 18:56:52     
Sep 16 18:56:52             sess.run([output_name], {input_name: x})
Sep 16 18:56:52         """
Sep 16 18:56:52         num_required_inputs = len(self._inputs_meta)
Sep 16 18:56:52         num_inputs = len(input_feed)
Sep 16 18:56:52         # the graph may have optional inputs used to override initializers. allow for that.
Sep 16 18:56:52         if num_inputs < num_required_inputs:
Sep 16 18:56:52             raise ValueError("Model requires {} inputs. Input Feed contains {}".format(num_required_inputs, num_inputs))
Sep 16 18:56:52         if not output_names:
Sep 16 18:56:52             output_names = [output.name for output in self._outputs_meta]
Sep 16 18:56:52 >       return self._sess.run(output_names, input_feed, run_options)
Sep 16 18:56:52 E       RuntimeError: Method run failed due to: [ONNXRuntimeError] : 2 : INVALID_ARGUMENT : Got invalid dimensions for input: 0 for the following indices
Sep 16 18:56:52 E        index: 0 Got: 8 Expected: 5
Sep 16 18:56:52 E        Please fix either the inputs or the model.
Sep 16 18:56:52 
Sep 16 18:56:52 ../.local/lib/python3.6/site-packages/onnxruntime/capi/session.py:72: RuntimeError

**Comment by houseroad** [comment]
@BowenBao could you take a look at the failing tests?

**Comment by BowenBao** [comment]
> @BowenBao could you take a look at the failing tests?

Thanks for catching that @houseroad, that appears to be a hidden bug with `test_arange`, which wasn't revealed with older version of onnxruntime. I'll send in a fix in a separate PR.

TLDR. The test tries to update exported onnx model to accept dynamic sized input, however it is written incorrectly such that the exported model input is still fixed sized. Meanwhile, the version of ort in CI doesn't validate if model input size matches with input data, so this error was not found.

**Comment by BowenBao** [comment]
Submitted PR https://github.com/pytorch/pytorch/pull/26320 for the CI failure. Will rebase this one once the fix is merged.
